### DISCUSSION TITLE
Is it possible one akvorado server support 2 ASN on akvorado.yaml?

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1439] [Q&A]
Hi Vincent / Expert,

May i know whether in one akvorado server we can assign two ASN in akvorado.yaml as per below? My uses case was i need to manage to ASN-1 (main company) and Subsidiary Company ASN-2.  If can, is it possible u guide me?

clickhouse:
  orchestrator-url: http://akvorado-orchestrator:8080
  kafka:
    consumers: 4
  servers:
    - clickhouse:9000
  prometheus-endpoint: /metrics
  asns:
    **64501: ACME Corporation**    -----------------> can we have 2 ASN and name?
  networks:

Thanks and appreciate your feedback

**Comment by vincentbernat** [comment]
This configuration is to customize the ASN names. You can put as many as you want.

**Reply by kronicklez** [reply]
Hi vincent,

Is it format below is correct?


asns:
        64501: ACME Corporation 
        64000: SingTel Corporation
 

Thanks and appreciate your feedback
